public/app/features/query/components/QueryEditorRow.tsx

```diff
@@ -57,6 +57,9 @@ interface Props<TQuery extends DataQuery> {
   history?: Array<HistoryItem<TQuery>>;
   eventBus?: EventBusExtended;
   alerting?: boolean;
+  onQueryCopied?: () => void;
+  onQueryRemoved?: () => void;
+  onQueryToggled?: (queryStatus?: boolean | undefined) => void;
```

Comment:
`queryStatus` is always passed to this callback so we can skip `| undefined`

```suggestion
  onQueryToggled?: (queryStatus: boolean) => void;
```

Reply:
It could have been casted to a boolean.